src/Razor/src/Microsoft.NET.Sdk.Razor/DiscoverMvcApplicationParts.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.AspNetCore.Razor.Tasks
+{
+    public class DiscoverMvcApplicationParts : Task
```

Comment:
How would you feel about removing the MVC terminology from this? There's nothing MVC in it.